Formula/pv.rb

```diff
@@ -20,9 +20,10 @@ class Pv < Formula
     sha256 "0c4d4a90c188370ed312490b7ff76fdb8a31399170cdc0ad5dfc1542af4c4fc0" => :yosemite
   end
 
+  patch :DATA
```

Comment:
Has this patch been submitted upstream? When can the patch be removed? Please put a (brief) comment before this line with information about this.

Reply:
Done!